pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html

```diff
@@ -610,7 +610,7 @@ <h3 class="modal-title">Verify LDAP Configuration</h3>
                                             <clr-icon aria-live="assertive" a11yTooltipTrigger shape="info-circle" size="22">
                                             </clr-icon>
                                             <clr-tooltip-content clrPosition="top-right" clrSize="lg" *clrIfOpen>
-                                                <span>A group name used to test LDAP group search functionality</span>
+                                                <span>A group name used to test LDAP group search functionality; can only be searched by OU.</span>
```

Comment:
I think in this case we can simplify to:
Enter the OU attribute for the group to test LDAP group search functionality.

Below on line 617, let's change the placeholder to OU